docs/server.md

```diff
@@ -96,6 +97,13 @@ This file descriptor must have already had the `bind(2)` and `listen(2)` system
 calls invoked on it. Additionally, it must be set non-blocking; try
 `fcntl(fd, F_SETFL, O_NONBLOCK)`.
 
+## Inspecting server state
+
+### server.getConnections(callback)
+
+The LDAP server API mirrors the node.js `server.getConnections` API. Callback
```

Comment:
Can you add a link to said documentation? LTS version should be fine.
```suggestion
The LDAP server API mirrors the [Node.js `server.getConnections` API](add link here). Callback
```

Reply:
Sure. I linked to the `latest-v12.x` version.